source/include/sigv4_internal.h

```diff
@@ -28,6 +28,12 @@
 #ifndef SIGV4_INTERNAL_H_
 #define SIGV4_INTERNAL_H_
 
+/* *INDENT-OFF* */
+#ifdef __cplusplus
```

Comment:
I don't think internal headers really need this

Reply:
Yeah, probably not, doesn't hurt though